python/scenario/voice/adapters/elevenlabs.py

```
@@ -223,6 +276,103 @@ async def disconnect(self) -> None:
+
+        if self._outbound_frames:
+            frame = self._outbound_frames.popleft()
+        elif self.awaiting_user_turn:
+            return
+        else:
+            frame = SILENCE_FRAME
+
+        try:
+            b64 = base64.b64encode(frame).decode()
+            await self._ws.send(json.dumps({"user_audio_chunk": b64}))
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)